packages/legacy/core/App/contexts/reducers/store.ts

```diff
@@ -646,6 +665,7 @@ export const reducer = <S extends State>(state: S, action: ReducerAction<Dispatc
       }
     }
     default:
+      console.log(`${action.type} is not available`)
```

Comment:
Can be helpful for local dev but shouldn't be committed, the linter will complain about `console` usage

Reply:
removed